upload_package

```diff
@@ -71,10 +71,19 @@ shopt -u nullglob
 
 echo "Copying update payload to update server"
 
-SERVER_UPDATE_DIR="/var/www/origin.release.flatcar-linux.net/update/${ARCH}/${VERSION}/"
 if [ "${NOUPLOAD}" = "" ]; then
-  ssh "core@${ORIGIN_SSH_URL}" mkdir -p "${SERVER_UPDATE_DIR}"
-  scp "${UPDATE_PATH}" "${UPDATE_CHECKSUM_PATH}" "${EXTRA_FILES[@]}" "${EXTRA_SUMS[@]}" "core@${ORIGIN_SSH_URL}:${SERVER_UPDATE_DIR}"
+  docker run --rm \
+    -v "${DATA_DIR}:/opt/data"
+    -v "${RCLONE_CONFIGURATION_FILE}:/opt/rclone.conf:ro" \
+    docker.io/rclone/rclone:1.71.1 \
+      --config "/opt/rclone.conf" \
+      sync \
+      # TODO: set '--dry-run' following 'NOUPLOAD' value.
+      --dry-run \
+      --progress \
+      # TODO: investigate the '--include' flag.
+      "/opt/data/" \
```

Comment:
So far other files in DATA_DIR were ignored - not sure what else would be lying around there but with a prefix replacement from DATA_DIR to /opt/data it should work to keep the current explicit list.